components/Sidebar/ChatHistory.tsx

```
@@ -161,79 +166,90 @@ export default function ChatHistory({
+                          if (chat.title === "Welcome to Study Chat") {
+                            router.push("/chat/welcome");
+                          } else if (chat.moduleId && chat.module) {
+                            const encodedName = encodeModuleSlug(
+                              chat.module.name
+                            );
+                            router.push(`/${encodedName}/chat/${chat.id}`);
+                          } else {
+                            router.push(`/chat/${chat.id}`);
+                          }
+                        }}
+                      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)